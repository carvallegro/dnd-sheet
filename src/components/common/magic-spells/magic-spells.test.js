import React from 'react'
import { shallow } from 'enzyme'

import MagicSpells from './index'

describe('<MagicSpells />', () => {
  it('renders', () => {
    const wrapper = shallow(<MagicSpells />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with props', () => {
    const wrapper = shallow(
      <MagicSpells spellLevel={1} slotExtended={2} slotTotal={3} />
    )
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with grid props', () => {
    const wrapper = shallow(<MagicSpells gridArea="testArea" columns={3} />)
    expect(wrapper).toMatchSnapshot()
  })
})
