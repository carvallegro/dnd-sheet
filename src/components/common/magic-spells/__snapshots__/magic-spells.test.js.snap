// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MagicSpells /> renders 1`] = `
<Styled(styled.div)
  columns={1}
>
  <SpellListHeading />
</Styled(styled.div)>
`;

exports[`<MagicSpells /> renders with grid props 1`] = `
<Styled(styled.div)
  columns={3}
  gridArea="testArea"
>
  <SpellListHeading />
</Styled(styled.div)>
`;

exports[`<MagicSpells /> renders with props 1`] = `
<Styled(styled.div)
  columns={1}
>
  <SpellListHeading
    slotExtended={2}
    slotTotal={3}
    spellLevel={1}
  />
</Styled(styled.div)>
`;
